### ISSUE TITLE
Question about automatic backup

### ISSUE THREAD

**Issue by dcapeletti** [teejee2008/timeshift#223]
Dear Ones, if I set up for an automatic weekly backup, reboot the system and I don't authenticate, does the system still back up without anyone being logged in?

**Comment by teejee2008** [comment]
Yes, the cron job runs as root so it doesn't require any users to be logged in

**Comment by dcapeletti** [comment]
Thank you for your answer. That is correct. It has been created automatically without any user having logged in. And by the way, weekly backups are done on Fridays.
